WTWSMS/common/religions/00_religions.txt

```diff
@@ -367,9 +367,11 @@ christian = {
 		can_grant_claim = yes
 		can_call_crusade = no
 		priests_can_inherit = no
-		max_wives = 4
+		max_wives = 1
```

Comment:
There's SOME evidence of the Ancient Gaels (Not at ALL the Ancient Britons) had some sort of Polygamy. But we're talking about the 1000's BC here, when they were still glorified hunter gatherers, and the Irish chieftain may have had a harem. That was long dead when the Romans arrived.

Though it's true the Irish practiced what you may call "pseudo-polygamy". They very often divorced (or killed) their wives for dumb reasons just to get a new one, which you can easily see when some Irish kings had six or seven wives in succession. Think of situations like the later Henry VIII. But even _that_ pseudo-polygamy is later period than the mod.